packages/mdc-tab-scroller/_mixins.scss

```diff
@@ -24,11 +24,36 @@
 @import "@material/tab/mixins";
 @import "@material/feature-targeting/functions";
 @import "@material/feature-targeting/mixins";
+@import "./variables";
+
+///
+/// Sets the CSS transition for the tab scrolling animation.
+///
+/// @param {Number | String} $duration-ms - Duration (in ms) of the animation.
+/// @param {String} $timing-function - Optionally overrides the default animation timing function.
+///
+@mixin mdc-tab-scroller-transition(
+  $duration-ms,
+  $timing-function: $mdc-animation-standard-curve-timing-function,
+  $query: mdc-feature-all()
+) {
+  $feat-animation: mdc-feature-create-target($query, animation);
+
+  &.mdc-tab-scroller--animating {
+    .mdc-tab-scroller__scroll-content {
+      @include mdc-feature-targets($feat-animation) {
+        transition: $duration-ms transform $timing-function;
+      }
+    }
+  }
+}
 
 @mixin mdc-tab-scroller-core-styles($query: mdc-feature-all()) {
   $feat-structure: mdc-feature-create-target($query, structure);
   $feat-animation: mdc-feature-create-target($query, animation);
 
+  @include mdc-tab-scroller-transition($mdc-tab-scroller-transition-duration, $query)
```

Comment:
This should be applied inside `.mdc-tab-scroller` selector.

Reply:
Done, thanks!